visualization/app/codeCharta/codeCharta.model.ts

```diff
@@ -255,8 +256,14 @@ export interface State {
 	appSettings: AppSettings
 	treeMap: TreeMapSettings
 	files: Files
+	lookUp: LookUp
 }
 
 export interface CCAction extends Action {
 	payload?: any
 }
+
+export interface LookUp {
+	pathToNode: Map<number, CodeMapNode>
+	pathToBuilding: Map<number, CodeMapBuilding>
```

Comment:
Damn, I totally forgot to rename the properties. it should be idToNode and idToBuilding now. Doing that asap. Don't merge yet! :D

Reply:
Done. :)